business_objects/export.py

```diff
@@ -29,7 +35,7 @@ def build_full_record_sql_export(
 
     if select_add:
         select_part += ",\n" + select_add
-    return get_final_sql(
+    return __get_final_sql(
```

Comment:
Such big sql assembly functions with lot of private functions seem to be hard to control. On first view, it seems to be ok by only sanitzing the project_id on the outer level here. I think we have also to consider the intermediate processing of user controllable values. For example: 1. query could select some labeling task and second query uses the labelink task name as variable in f string. Then we would still need to sanitze the column name I would assume.

Reply:
understandable, ill put it under further extensions in the PR.
I think the most common case for that is the attributes name as it is the only one that is used in queries directly (not select but as access). (Used e.g. in the database as the json value selection in postgres needs the name)